src/app/login/login.component.scss

```
@@ -580,117 +185,192 @@
   }
 }
 
-// ===== Resources Section =====
-.resources-section {
-  margin-top: auto;
-  padding-top: 1rem;
-  flex-shrink: 0;
-  animation: fade-in-up 0.6s ease-out 0.4s both;
-}
+// ===== Override form field styles for clean rounded inputs =====
+::ng-deep {
```

Stylelint (17.13.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)